### ISSUE TITLE
Building 'Cryptodome.Hash._ghash_clmul' extension fails on SLES 11.

### ISSUE THREAD

**Issue by dumol** [Legrandin/pycryptodome#338]
Hi again,

This is a problem I've had with PyCryptodome 3.6.6 as well, but failed to report it. Have used your wheel in the meantime with no issues.

I'm retesting with 3.9.1 now and still see the same issue (I think), despite the seemingly related fix in https://github.com/Legrandin/pycryptodome/issues/232.

Here's how things look on my side (SUSE Linux Enterprise Server 11.4 with GCC 4.3.4):

```
    building 'Cryptodome.Hash._ghash_clmul' extension
    gcc -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -fPIC -DHAVE_STDINT_H -DPYCRYPTO_LITTLE_ENDIAN -DSYS_BITS=64 -DLTC_NO_ASM -DHAVE_UINT128 -DHAVE_CPUID_H -DHAVE_POSIX_MEMALIGN -Isrc/ -I/srv/buildslave/runtime/build-sles11-x64/slave/python-package-sles-11/build/build/python2.7-lnx-x64/include/python2.7 -c src/ghash_clmul.c -o build/temp.linux-x86_64-2.7/src/ghash_clmul.o -mpclmul -mssse3 -O3
    src/ghash_clmul.c:41:2: error: #error Either intrin.h or x86intrin.h header files must be available
    src/ghash_clmul.c: In function 'swap':
    src/ghash_clmul.c:179: warning: implicit declaration of function '_mm_shuffle_epi8'
    src/ghash_clmul.c:179: error: incompatible types in return
    error: command 'gcc' failed with exit status 1
```

Just wanted to let you know… It's no big deal to keep using your wheel on this platform. Thanks!

**Comment by Legrandin** [comment]
Thanks for reporting. Fixed in version 3.9.2.

**Comment by dumol** [comment]
Can confirm, building works now on SLES 11 and tests pass. 

As usual, excellent work, thank you!
